gandalf.js

```diff
@@ -10,125 +10,119 @@
  * Released under the MIT license.
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.